### DISCUSSION TITLE
When I upgrade 1.2.5 to 1.2.7, I lost my pools

### DISCUSSION THREAD

**Discussion by cccat6** [Chia-Network/chia-blockchain#8511] [Support]
I upgrade my Chia full_node from 1.2.5 to 1.2.7 yesterday. Once it has been done, I see it farming regularly.
However, I find I receive no credit from pools. When I check my Chia client, I find that the Pool list is empty but the Wallet is synced and shows my transportation correctly.
I don't know why. The only thing I can do is delete the wallet db file and resync it.

**Comment by loppefaaret** [comment]
so having resynced your wallet database, you now have all your plotNFT's listed again ?

**Reply by cccat6** [reply]
Yes it does

**Reply by loppefaaret** [reply]
thank you for reporting back

**Comment by BrandtH22** [comment]
@cccat6 , since your last comments indicates the pool information is back after a resync it seems this issue has been resolved.
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
